app/permission/rest.py

```diff
@@ -10,17 +14,12 @@
 
 @permission.route('', methods=['GET'])
 def get_permissions():
-    data, errors = permission_schema.dump(
-        permission_dao.get_query(filter_by_dict=request.args), many=True)
-    if errors:
-        abort(500, errors)
+    data = permission_schema.dump(permission_dao.get_query(filter_by_dict=request.args), many=True).data
```

Comment:
I assumed 500 was an error in this case as any real 500 in this case I guess would be db related and we have error handlers for those.